### PR TITLE
891 update to v1.8.0

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Comment by SandyEma9** [comment]
@robert-altom @RobertPoienar @gomboskriszta15 @ReichEdvin @iulianatodoran

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,32 +0,0 @@
-stages:
```
Comment (ReichEdvin) [inline .gitlab-ci.yml:1]: You should add a workflow to run the tests, you can use https://game.ci/docs/github/getting-started
If you want you can create a new issues to add workflow

File: README.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
 # Android Build with CSharp Tests
 
-This repository shows a few C# tests that use the page object model and AltUnityTester to test the Unity endless runner sample:
+This repository shows a few C# tests that use the page object model and AltTester to test the Unity endless runner sample:
```
Comment (ReichEdvin) [inline README.md]: I think it's AltTester Unity SDK

**Review by ReichEdvin — APPROVED** [review]
LGTM

**Review by iulianatodoran — APPROVED** [review]
tested locally and LGTM

**Pull request merged**